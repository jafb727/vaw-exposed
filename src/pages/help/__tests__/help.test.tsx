/**
 * @author Jose A Felix
 * @name help.test
 * @description Help page component unit testing file
 */

/** -------------------------------------------------------- */

/** @import Components */
import Help from "..";

/** @import Utilities */
import { renderComponent } from "../../../test/setupTests";

/** -------------------------------------------------------- */

describe("Help", () => {
   it("Renders ok", () => {
      /** Component context mounting */
      const component = renderComponent(<Help />);

      /** Main assertion */
      expect(component).not.toBe(null);
   });

   it("Renders main component wrapper", () => {
      /** Component context mounting */
      const { getByTestId } = renderComponent(<Help />);

      /** Main assertion */
      const component = getByTestId("help-page");
      expect(component).toBeInstanceOf(HTMLElement);
   });

   it("Matches snapshot", () => {
      /** Component context mounting */
      const { getByTestId } = renderComponent(<Help />);

      /** Main assertion */
      const component = getByTestId("help-page");
      expect(component).toMatchSnapshot();
   });
});
