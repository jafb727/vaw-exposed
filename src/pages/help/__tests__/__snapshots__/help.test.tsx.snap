// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Help Matches snapshot 1`] = `
<div
  data-testid="help-page"
>
  Help
</div>
`;
